### ISSUE TITLE
BackgroundColor of a Label doesn't work in Release on iOS

### ISSUE THREAD

**Issue by Hooterr** [dotnet/maui#16250]
### Description

I guess we have another `IsVisible` type of bug...
`BackgroundColor` on a `Label` doesn't work in release on iOS.

It doesn't work through binding
It doesn't work in CollectionView template
It doesn't work with DataTrigger
It doesn't work with styles
It doesn't work at all.

What the actual f... are you guys doing over there?!

https://github.com/dotnet/maui/assets/28758721/91a92064-a323-4e67-91f2-bbc2b7cfc014

### Steps to Reproduce

```
Label.BackgroundColor = Colors.Red;
Label.Text = "Red";
```

### Link to public reproduction project repository

https://github.com/Hooterr/BackgroundColorLabelBugMaui

### Version with bug

7.0.86 and 7.0.92

### Last version that worked well

Unknown/Other

### Affected platforms

iOS

### Affected platform versions

iOS 16.5.1

### Did you find any workaround?

Use `Background`.

But cmon. What the hell.....

**Comment by Hooterr** [comment]
I guess doesn't work an android either #16220

**Comment by acaliaro** [comment]
If you change X:Name Label with another name ex label with lowercase l?

**Comment by Hooterr** [comment]
> If you change X:Name Label with another name ex label with lowercase l?

No, it doesn't work either.

**Comment by hartez** [comment]
Which version of VS do you have installed? I just tested your repro against service release 7 and it worked fine in Debug and in Release. 

7.0.86 is definitely broken for both Release _and_ Debug.

**Comment by mattleibow** [comment]
I think this is the AOT issue:

 - https://github.com/dotnet/maui/issues/11662

Adding this to your maui program builder will make it work:

```cs
.ConfigureMauiHandlers(_ => 
{
	LabelHandler.Mapper.AppendToMapping(
		nameof(View.BackgroundColor),
		(handler, View) => handler.UpdateValue(nameof(IView.Background)));
})
```

Same thing happens with Button. We have a chain of things that need to land, but this should fix it for now.

**Comment by mattleibow** [comment]
Duplicate of #11662

**Comment by Hooterr** [comment]
What? Another bug from Nov 2022 and you mark this as duplicate? How is this not fixed?? How can we know if other properties are going to stop working again?? Should we do this workaround for every control that has a background color?? 
What you posted is just a workaround. This is NOT resolved!

**Comment by hartez** [comment]
> What? Another bug from Nov 2022 and you mark this as duplicate? 

Yes, the bug you're seeing has the same root cause as #11662. So we're tracking everything there. 

> How hasn't this been fixed yet?? 

Because it took a long time to diagnose accurately and a longer time to fix (https://github.com/dotnet/maui/pull/13836 - this entire list of PRs had to land to fix this correctly).

> How can we know if other properties are going to stop working again?? 

We've added tests (both automated and manual) to catch this sort of thing. 

> Should we do this workaround for every control that has a background color?? 

No. It doesn't affect every control. 

> What you posted is just a workaround. 

Yes, we've posted a workaround you can use _until_ the bug is resolved. 

> This is NOT resolved!

Correct. The original report of this bug is still open, and will be resolved soon (via #13836). _This_ issue is now linked to the main issue.
